go-getting-started/controllers/find_one.go

```diff
@@ -0,0 +1,13 @@
+package controllers
+
+import (
+	"context"
+
+	"github.com/heroku/go-getting-started/dbinterface"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+func FindId(collection dbinterface.CollectionAPI, query map[string]interface{}) *mongo.SingleResult {
+	res := collection.FindOne(context.TODO(), query)
```

Comment:
Please pass the context 'from above' instead of creating it here.
http://p.agnihotry.com/post/understanding_the_context_package_in_golang/